### DISCUSSION TITLE
Postprocessor Output

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#21404] [Q&A General] [answered]
Hello,

I have a calculation with 10.000 timesteps in the course of 10 seconds. I want to output data with a Postprocessor but only at certain times (0, 0.2, 0.6, 1, 2, 3, 4, 6, 10 seconds) and not at every timestep because that would result in a huge amount of irrelevant .csv documents. Is there a way to do this in MOOSE?

**Comment by Flolaffel** [comment, marked as answer]
I found the solution. In the Outputs-Block use

```
[csv]
  type = CSV
  sync_times = '0.2 0.6 ...'
  sync_only = true
[]
```

**Reply by mangerij** [reply]
useful!
